gym_ignition/base/robot/robot_links.py

```diff
@@ -26,19 +26,55 @@ def link_names(self) -> List[str]:
         """
 
     @abstractmethod
-    def link_pose(self, link_name: str) -> np.ndarray:
+    def link_pose(self, link_name: str) -> Tuple[np.ndarray, np.ndarray]:
         """
         Return the pose of the specified link.
 
         The pose is composed of a position and a quaternion associated to the link frame,
-        both expressed in the the robot base frame.
+        both expressed in the the world frame.
 
         Args:
             link_name: The name of the link
 
         Returns:
-            A Tuple containing the position and orientation of the base:
+            A Tuple containing the position and orientation of the link:
 
             - position: 3D array in the [x, y, z] form.
             - orientation: a 4D array containing a quaternion in the [w, x, y, z] form.
         """
+
+    @abstractmethod
+    def link_velocity(self, link_name: str) -> Tuple[np.ndarray, np.ndarray]:
+        """
+        Return the velocity of the specified link.
+
+        The velocity is composed of linear and angular velocities of the link frame,
+        both expressed in the world frame.
```

Comment:
In which frame the derivation is done?

Reply:
How would you modify this description to be unambiguous? For the future, we still have #37 pending.

Reply:
I think it is difficult to specify it non-ambitiously with word only in a self-contained way, ideally it would be good to have a separate short piece of docs to define the quantities using formulas, but this is probably something that could be part of https://github.com/robotology/gym-ignition/issues/37 indeed.  
Assuming that this velocity is the mixed one as defined in Equation 35 of https://research.tue.nl/files/139293126/A_Multibody_Dynamics_Notation_Revision_2_.pdf, to minimize ambiguity I would write: 
The velocity is composed of linear velocity (first three elements) and angular velocity (last three elements) of the link frame.
The linear velocity is the time derivative of the position of the origin of the link frame expressed in the world frame. 
The angular velocity is expressed with the orientation of the world frame.

Reply:
Updated documentation in https://github.com/robotology/gym-ignition/pull/117/commits/dfb556714c5bf600a6b64ae2748df5883d8e89fa. If this looks fine, I will proceed merging this PR.

Reply:
> Updated documentation in [dfb5567](https://github.com/robotology/gym-ignition/commit/dfb556714c5bf600a6b64ae2748df5883d8e89fa). If this looks fine, I will proceed merging this PR.

I think now it is clear.